tac/project-updates/2025/2025-annual-Identus.md

```diff
@@ -0,0 +1,91 @@
+---
+layout: default
+title: 2025 Annual Review Identus
+parent: 2025
+grand_parent: Project Updates
+---
+
+## Project Health
+
+LF Decentralized Trust Identus [LFX Insights Dashboard](https://insights.lfx.linuxfoundation.org/foundation/lf-decentralized-trust/reports/contributors?project=identus&bestPractice=false&dateFilters=Last%20Year&dateRange=2024-01-01%20to%202024-12-31&compare=PP&granularity=month&hideBots=true&repository=all)
+The project is relatively new to the LF Decentralized Trust Ecosystem and is mainly supported by the IOG maintainers.
+At the same time, over the past year, we have seen several new contributors and evidence of usage of the Identus platform by other companies: 
+- https://socious.io/, Socious production use case.
+- https://ahau.io/, Ahau PoC.
+- https://blocktrust.dev/, - identity wallet.
+
+We currently utilize two Discord servers: the LF Decentralized Trust Discord and the private Identus Discord server. 
+The Identus Discord has seen significantly more engagement. 
+Since January 2024, our membership has grown from 258 to 476 members. 
+On average, there are 348 activities across various channels monthly, with about 39 users online daily.
+
+## Maintainer Diversity
+
+There are currently 10 maintainers from IOG and other companies. 
+
+The MAINTAINERS.md file is up to date and can be found [here](https://github.com/hyperledger/identus/blob/main/MAINTAINERS.md)
+The `identus-maintainers` [team](https://github.com/hyperledger/governance/blob/main/access-control.yaml#L625-L655) includes around 10 members from IOG and other companies.
+
+The number of active maintainers has decreased due to restructuring the Atala Team at IOG. 
+However, the diversity of maintainers has increased over the past six months, with more companies contributing to the project.
+
+## Project Adoption
+
+Several companies are using the Identus project for prototyping and production use cases.
+
+The following companies are using the project:
+- https://socious.io/, Socious production use case.
+- https://ahau.io/, Ahau PoC.
+- https://blocktrust.dev/, - identity wallet.
```

Comment:
This list seems redundant. You can just add a reference to the "Project Health" section. Alternatively, you can refer to this list from the "Project Health" section.